includes/admin/payments/orders.php

```diff
@@ -33,6 +33,21 @@ function edd_order_details_publish( $order ) {
 		<div class="edit-post-header">
 
 			<div class="edit-post-header__settings">
+				<?php if ( edd_is_add_order_page() ) : ?>
+					<div class="edd-send-purchase-receipt">
+						<div class="edd-form-group">
+							<div class="edd-form-group__control">
+								<input type="checkbox" name="edd_order_send_receipt" id="edd-order-send-receipt" class="edd-form-group__input" value="1" />
```

Comment:
If this is supposed to be checked on page load (I think that's how the issue suggests), it needs to include `checked` as an attribute (just `checked`, doesn't `=` anything ([ref](https://www.w3schools.com/tags/att_input_checked.asp))

Reply:
Good catch!